### PR TITLE
add ut for pkg/slices/slices_test.go

### PR DESCRIPTION
**Pull request by hexiaodai**
**Please provide a description of this PR:**

add ut for pkg/slices/slices_test.go

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [X] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/slices/slices_test.go [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+		{
+			name:      "equal",
+			elements:  []string{"a", "b"},
+			elements2: []string{"a", "b"},
+			want:      true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := Equal(tt.elements, tt.elements2); got != tt.want {
```
Comment (howardjohn) [inline pkg/slices/slices_test.go:57]: this is just an alias for the upstream package, I am not sure we need to test it?

Comment (hexiaodai) [reply pkg/slices/slices_test.go]: > this is just an alias for the upstream package, I am not sure we need to test it?

I removed the test method for the alias function

**Pull request merged**